packages/react-native-codegen/src/parsers/flow/parser.js

```diff
@@ -25,9 +25,6 @@ class FlowParser implements Parser {
     switch (propertyOrIndex.type) {
       case 'ObjectTypeProperty':
         return propertyOrIndex.key.name;
-      case 'ObjectTypeIndexer':
```

Comment:
By removing this, if we end up in this case, we are going to throw an error. I think that this change makes impossible to end up in this case, am I right? (Just double-checking to make sure we don't miss anything.)

Reply:
It won't, because it has been recognized as a dictionary object before and will never go into this function.

And overall, indexer **IS NOT** a property, so it couldn't be processed here anyway.